src/test/java/MainTest.java

```diff
@@ -0,0 +1,205 @@
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import uk.ac.ebi.pride.sdrf.validate.model.ValidationError;
+import uk.ac.ebi.pride.sdrf.validate.util.Constants;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * @author Suresh Hewapathirana
+ */
+@Slf4j
+class MainTest {
+
+    String templateName = "default";
+    boolean verbose = true;
+
+    /**
+     * Validate example sdrf file in the resource folder
+     *
+     * @throws Exception
+     */
+    @Test
+    void validateSampleFile() throws Exception {
+
+        URL url = MainTest.class.getClassLoader().getResource("sdrf.txt");
+        if (url == null) {
+            throw new IllegalStateException("no file for input found!");
+        }
+        File sdrfFile = new File(url.toURI());
+
+        Constants.Templates template = Enum.valueOf(Constants.Templates.class, templateName.trim().toUpperCase());
+        List<ValidationError> errors = Main.validate(sdrfFile.getAbsolutePath(),template, verbose);
+        Assertions.assertTrue(errors.size()==0);
+    }
+
+    /**
+     * Run a bulk test with the sdrf files in the annotated projects in Github repository
+     * @throws Exception
+     */
+    @Test
+    void bulkValidation() throws Exception {
+
+        final String repoURL = "https://raw.githubusercontent.com/bigbio/proteomics-metadata-standard/master/annotated-projects/";
+        Set<String> accessions = new HashSet<>(Arrays.asList(
```

Comment:
@sureshhewabi I don't think this is a good idea. We should validate the datasets manually.

Reply:
I thought this will be easy to check the correctness of the code, when we modify the code. I already tested all the files, and test case got passed. Therefore, shall I remove it completely or just add ignore annotation to the test case?

Reply:
Another way of doing it, is to produce a jar tool and in the github actions you can download all the sdrfs from the other repository and then validate all of them. The way is now implemented you need to add all the time new accessions in the code every time a new dataset is added.

Reply:
I will try that. Thanks